plasmapy/simulation/particle_integrators.py

```diff
@@ -36,6 +36,69 @@ def boris_push(x, v, B, E, q, m, dt):
     dt : `float`
         Timestep, in SI (second) units.
 
+    inplace : 'bool'
+        If True, v and x array are changed with boris_push, default False.
```

Comment:
Expanded on this a little:
```suggestion
        If set, data in the v and x arrays is changed during execution.
        Defaults to True for performance reasons.
        If False, this function returns `None`.
```